lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -246,13 +246,17 @@ private function getLastModification(ReflectionClass $class)
         ));
     }
 
+    /**
+     * @param ReflectionClass $reflectionTrait
+     * @return int
+     */
     private function getTraitLastModificationTimes(ReflectionClass $reflectionTrait)
```

Comment:
This method returns an `int[]`, which is then passed to `max()` in `private function getLastModification()`. If I get it correctly, the fix is different, and it is to change that function's body instead:

```php
         return max(array_merge(
             [$filename ? filemtime($filename) : 0],
            // notice the three dots:
             ...array_map([$this, 'getTraitLastModificationTimes'], $class->getTraits()),
             array_map([$this, 'getLastModification'], $class->getInterfaces()),
             $parent ? [$this->getLastModification($parent)] : []
         ));
```